File: 151002/DTalalaev/RV/rv3/src/main/java/by/poit/dtalalaev/entity/PostResponseTo.java
package by.poit.dtalalaev.entity;

import lombok.*;

import javax.validation.constraints.Size;
import java.math.BigInteger;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class PostResponseTo {
    private BigInteger id;
    private BigInteger storyId;
    @Size(min =2 , max = 2048, message = "Content must be between 2 and 2048 characters")
    private String content;

    public PostResponseTo(Post post) {
        this.id = post.getId();
        this.storyId = post.getStoryId();
        this.content = post.getContent();
    }
}

File: 151004/Ermolovich/Dc_Lab4/discussion/DC_Lab1/DTO/PostResponseTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class PostResponseTo(int Id, int? storyId, string? Content) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public int? storyId { get; set; } = storyId;

        public string? Content { get; set; } = Content;

    }
}
